pulsar-broker/src/main/java/org/apache/pulsar/PulsarClusterMetadataSetup.java

```diff
@@ -475,10 +490,13 @@ public static MetadataStoreExtended initLocalMetadataStore(String connection, in
         return store;
     }
 
-    public static MetadataStoreExtended initConfigMetadataStore(String connection, int sessionTimeout)
```

Comment:
Is it useful to keep backwards compatibility by keeping this method signature and delegating to the new method signature with `configPath` set to `null`? Just thinking of possible cases where this class is used by some external code.